server/neptune/gateway/event/push_event_handler.go

```diff
@@ -39,14 +43,21 @@ type scheduler interface {
 	Schedule(ctx context.Context, f sync.Executor) error
 }
 
-const defaultWorkspace = "default"
+const DefaultWorkspace = "default"
 
 type PushHandler struct {
-	Allocator      feature.Allocator
-	Scheduler      scheduler
-	TemporalClient signaler
-	Logger         logging.Logger
-	GlobalCfg      valid.GlobalCfg
+	Allocator                     feature.Allocator
+	Scheduler                     scheduler
+	TemporalClient                signaler
+	Logger                        logging.Logger
+	GlobalCfg                     valid.GlobalCfg
+	ProjectFinder                 events.ProjectFinder
+	VCSClient                     vcs_client.Client
```

Comment:
Again, let's not use this if we can avoid it.  Just create an interface on this side with the method you need.